libs/platform/src/lib/components/menu/menu-component/menu-item.component.ts

```diff
@@ -52,6 +52,7 @@ export class MenuItemComponent implements OnInit, DefaultMenuItem, OnChanges {
     public childItems: MenuItem[] = [];
 
     /** calculates the final width of the label when icons are used */
+    /** @hidden */
     public finalItemWidth = '';
 
     @Output() itemClick: EventEmitter<void> = new EventEmitter();
```

Comment:
I would recommend to make it readonly and follow our code style like 
`@output()`
`
itemClick: ...
`

Reply:
done